build.gradle

```diff
@@ -1,11 +1,10 @@
 plugins {
     id 'nu.studer.plugindev' version '1.0.3'
     id 'com.jfrog.bintray' version '1.2'
-    id 'com.ullink.msbuild' version '2.18'
-    id 'com.ullink.nuget' version '2.15'
+    id 'com.ullink.msbuild' version '2.12'
```

Comment:
should not rollback those gradle plugin?
I think it could be the failure reason

Reply:
Yes, was unintended